service-perfplat/docker/Dockerfile-worker-ci

```diff
@@ -17,6 +17,10 @@ FROM public.ecr.aws/lambda/python:3.8
 
 # fixes security issues detected in build.
 RUN yum update -y rpm openssl curl
+RUN yum update nss
```

Comment:
combine these with the other ones above as this will then create 1 layer on docker rather than 4 extra ones.